### DISCUSSION TITLE
When will NFT plot support be possible

### DISCUSSION THREAD

**Discussion by jeus1609** [ericaltendorf/plotman#835] [Q&A]

**Comment by altendky** [comment]
Already done.  Will be released soon.  Note that plotman releasing support is not the same as Chia Network releasing the actual pooling protocol as finalized, wait for them to start replotting.

**Reply by TaylorM87** [reply]
Hazaa, Chia 1.2 has released which means pools are now considered "Live". I can't wait until Plotman is released. I have saved about 200TB to plot to pools. Thanks for all that you devs do here.

**Reply by ahansen1** [reply]
I was looking for the branch that it's done on, but wasn't able to find one.  Has anyone else found it?

**Reply by altendky** [reply]
v0.5 was released yesterday.

**Reply by ahansen1** [reply]
I'm on 0.5, but wasn't seeing where to configure the "-c" value in the config.  I must have overlooked something.  I'll dig back in.

**Reply by altendky** [reply]
`-c` is the pool contract address (note the `--help` for `chia plots create` shows it as `-c, --pool_contract_address`).  https://github.com/ericaltendorf/plotman/wiki/Configuration#2-v05  and yes, we need a schema/docs for the configuration file for plotman.

**Reply by ahansen1** [reply]
I see it now!  Thanks!  I had just installed fresh and done an init and was going off the generated config looking for the option.  I should have searched the code earlier but had found this thread and was thinking it wasn't in yet.  Sorry to be a bother.  Thanks for your super prompt responses!

**Reply by TaylorM87** [reply]
@altendky Is there a way to verify that this is making a portable plot vs a normal plot? I followed the config file and placed in the pool_contract_address and the farmer_pk respectively and plotman started a plot but I can't tell if it's creating the correct type of plot.

**Reply by altendky** [reply]
You could `ps aux | grep 'plots create'` and look at the passed options to see if they are what you want.  Though, this does strike me as a new discussion.

**Reply by TaylorM87** [reply]
Very nice it's showing -f and -c respectively and is working. The default plotman.yaml that is generated doesn't include the pool_contract_address field to input and also had to delete the # and then another backspace to get the format correct on the yaml. Thank you for the quick reply.

**Reply by altendky** [reply]
The default yaml is being worked on.  https://github.com/ericaltendorf/plotman/pull/839

**Reply by ladiaz1028** [reply]
where do I replace -p to -c

**Reply by baron-orangutang** [reply]
get this exception when I specify the farmer_pk & pool_contract_address in my config.yaml.  

marshmallow.exceptions.ValidationError: {'plotting': {'chia': {'pool_contract_address': ['Unknown field.'], 'farmer_pk': ['Unknown field.']}}}

My config is as below withthe 'xxxx's replaced with the correct keys. 


plotting:
        # Your public keys: farmer and pool - Required for madMAx, optional for chia with mnemonic.txt
        # farmer_pk: ...
        # pool_pk: ...

        # If you enable Chia, plot in *parallel* with higher tmpdir_max_jobs and global_max_jobs
        type: chia
        chia:
                # The stock plotter: https://github.com/Chia-Network/chia-blockchain
                k: 32                # k-size of plot, leave at 32 most of the time
                e: False             # Use -e plotting option
                n_threads: 2         # Threads per job
                n_buckets: 128       # Number of buckets to split data into
                job_buffer: 3389     # Per job memory
                farmer_pk: xxxxx
                pool_contract_address: xxxxx


        # If you enable madMAx, plot in *sequence* with very low tmpdir_max_jobs and global_max_jobs
        madmax:
                # madMAx plotter: https://github.com/madMAx43v3r/chia-plotter
                n_threads: 4          # Default is 4, crank up if you have many cores
                n_buckets: 256        # Default is 256

**Reply by altendky** [reply]
Please do not just tack on follow up questions to discussions that have already been answered.  Also, look at the previous responses and links.
